wear/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -0,0 +1,94 @@
+package chat.rocket.android.main.ui
+
+import android.app.Activity
+import android.app.Fragment
+import android.os.Bundle
+import androidx.wear.widget.drawer.WearableNavigationDrawerView
+import chat.rocket.android.R
+import chat.rocket.android.main.presentation.MainPresenter
+import chat.rocket.android.main.presentation.MainView
+import com.google.firebase.FirebaseApp
+import com.google.firebase.iid.FirebaseInstanceId
+import com.google.firebase.messaging.FirebaseMessaging
+import dagger.android.*
+import kotlinx.coroutines.experimental.CommonPool
+import kotlinx.coroutines.experimental.launch
+import timber.log.Timber
+import javax.inject.Inject
+
+
+class MainActivity : Activity(), MainView, HasActivityInjector,
```

Comment:
If one of the functions with `TODO(something)` is being called then the app crashes. Could you fix these TODO's?